docs/usage.md

```diff
@@ -273,22 +275,22 @@ DROP - aberrant splicing
+To build your own database you will need at least 50 samples for aberrant expression, if you only run aberrant splicing 30 samples will suffice but DROP authors recommend to have at least around 100 for both modules. You can use Tomte to build your own database, to do so we recommend to run with the following parameters:
 
-- `--skip_export_counts_drop false` this switch will ensure that a folder called export_counts is created
+- `--skip_export_counts_drop false` this switch will ensure that a folder in references called export_counts is created
 - `--skip_drop_as false` if you want to get a database for aberrant splicing
 - `--skip_drop_ae false` if you want to get a database for aberrant expression
 - `--skip_subsample_region false` if you have sequenced any material with overrepresented regions (such as hemoglobin in whole blood) we recommend to remove it by setting this parameter to false and providing a bed with the overrepresented region with `--subsample_bed`
-- `--skip_downsample false` if you have very deeply sequenced samples, we recommend to downsample, the default is 60M read pairs
-- `--skip_build_tracks true`, `--skip_stringtie true`, `--skip_variant_calling`, `--skip_vep true` as most users will be interested in getting the database rather than other downstream results
+- `--skip_downsample false` if you have very deeply sequenced samples, we recommend to downsample, the default is 60M read pairs (or 120M reads).
+- `--skip_build_tracks true`, `--skip_stringtie true`, `--skip_variant_calling`, `--skip_vep true` as most users will be interested in getting the database rather than other downstream results, which will require a considerable amount of resources given the number of samples run.
 
-Running DROP with many samples requires a lot of time and a lot of memory, that is why we recommend to subsample overrepresented regions and downsample if you have deeply sequenced samples. If your run fails for either of this reasons, try to relaunch it from the work directory where DROP was run so that DROP continues from the point where it failed (if you restart the pipeline with `-resume` it will begin from the start and it will likely fail in the same way).
+Running DROP with many samples requires a lot of time and a lot of memory, that is why we recommend to subsample overrepresented regions and downsample if you have deeply sequenced samples. If your run fails, we recommend increasing memory without increasing the number of cores. If it fails for and has been running for a while, try to relaunch it from the work directory where DROP was run so that DROP continues from the point where it failed (if you restart the pipeline with `-resume` it will begin from the start).
```

Comment:
```suggestion
Running DROP with many samples requires a lot of time and a lot of memory, that is why we recommend to subsample overrepresented regions and downsample if you have deeply sequenced samples. If your run fails, we recommend increasing memory without increasing the number of cores. If it fails and has been running for a while, try to relaunch it from the work directory where DROP was run so that DROP continues from the point where it failed (if you restart the pipeline with `-resume` it will begin from the start).
```